run_unittests.py

```diff
@@ -5607,6 +5607,26 @@ def test_identity_cross(self):
         # TODO should someday be explicit about build platform only here
         self.init(testdir, override_envvars=env)
 
+    def test_static_link(self):
+        if is_cygwin():
+            raise unittest.SkipTest("Cygwin doesn't support LD_LIBRARY_PATH.")
+
+        # Build some libraries and install them
+        testdir = os.path.join(self.unit_test_dir, '69 static link/lib')
+        libdir = os.path.join(self.installdir, self.prefix[1:], self.libdir)
+        self.init(testdir)
+        self.install()
+
+        # Test that installed libraries works
+        self.new_builddir()
+        testdir = os.path.join(self.unit_test_dir, '69 static link')
+        env = {'PKG_CONFIG_LIBDIR': os.path.join(libdir, 'pkgconfig')}
+        run_env = {'LD_LIBRARY_PATH': libdir}
+        self.init(testdir, override_envvars=env,
+            extra_args=['-Dc_link_args="-L{}"'.format(libdir)])
```

Comment:
__[Flake8]__ [E128] continuation line under-indented for visual indent ([link](https://sider.review/gh/repos/19784232/pulls/5936#issue-4627294))

<sub>posted by Sider</sub>